.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Terraform Test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Integration test
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: terraform validate
+        uses: dflook/terraform-validate@v1
+
+      - name: terraform fmt
+        uses: dflook/terraform-fmt-check@v1
+
+      - name: terraform test
+        uses: dflook/terraform-test@v1
+        env:
+          CF_USER: ${{ secrets.CF_USER }}
```

Comment:
[question (nb)] it seems like we usually do `CF_USERNAME`, is there a reason to go with the shorter here?

Reply:
p.s. I see the shorter version was in here before so maybe it isn't relevant to this PR, though I remain curious if you do know a reason.

Reply:
because naming things is one of the hard problems

we mostly use `CF_USERNAME` in a lot of places because cf-cli will pick that up automatically during an argument-less `cf auth`

`CF_USER` is used here because that's what the terraform provider will pick up automatically for an argument-less configuration

Reply:
Ahhhhh didn't know that about the terraform provider, thanks.